management-commands/config.md

```diff
@@ -1,7 +1,16 @@
 # config
 
-The `config` command allows you to configure various settings for the server, such as roles, channels, reaction roles,
-and settings.
+The `config` command allows you to configure all settings for the server, such as roles, channels, reaction roles,
+and other customizable behavior of MoonlightBot. To get the current configuration of a setting, just use the
+corresponding sub command without any optional arguments.
+
+{% hint style="info" %}
+If an argument provides you with the option `Open Editor` like the argument `custom-notification-text` of
+the [`roles on-expire`](../management-commands/config.md#on-expire) command, it means that a modal textbox will open for
+you to input your desired text.
+<br>
+All default values are either `0`,`empty` or `false` for their respective arguments, unless stated otherwise.
```

Comment:
```suggestion
All default values are either `0`, `false` or empty for their respective arguments, unless stated otherwise.
```

"empty" in itself isn't a value